src/util/parse-options-test.cc

```diff
@@ -120,7 +120,7 @@ void UnitTestParseOptions() {
     po4.Register("option", &val, "My boolean");
     po4.Read(argc4, argv4);
     assert(false); // Should not reach this part of code.
-  } catch(std::exception e) {
+  } catch(std::exception &e) {
```

Comment:
I suggest using `const std::exception &e`.

`const` makes the intention clear that it is not intended to be modified.